tls/listener.go

```diff
@@ -31,6 +31,11 @@ import (
 	"time"
 )
 
+const (
+	// same as golang Dial default
+	keepAlive = 15 * time.Second
```

Comment:
Is this something that we will want to configure?

Reply:
I doubt it since we hadn't so far. And also downstream handlers can override these settings if they prefer